che-incubator/che-code
# fix: Include the commands imported from a parent Devfile as to VS Code Tasks

### What does this PR do?
When Che-Code translates the Devfile Commands to VS Code Tasks, it ignored the commands defined in a parent Devfile. With this patch, parent Commands are imported now.

### What issues does this PR fix?

addresses the https://github.com/che-incubator/che-code/pull/194#discussion_r1151103256
fixes https://github.com/eclipse/che/issues/22373

### How to test this PR?
1. Spin up a new workspace from the test project https://github.com/azatsarynnyy/inventory-quarkus/tree/with-parent which is configured to use the Che-Code image built from this PR.
2. Open VS Code Tasks list and verify that the commands defined in the parent Devfile are listed as well as the commands defined in the project's Devfile:
![image](https://github.com/che-incubator/che-code/assets/1636395/cbfa721d-22c6-4488-b170-0585f9acfb5b)
